src/HomeKit/HMHome.cs

```diff
@@ -87,57 +90,40 @@ public partial class HMHome
 
 #if !XAMCORE_4_0
 
-#if !NET
+#if NET
+		[SupportedOSPlatform ("ios8.0")]
+		[SupportedOSPlatform ("maccatalyst14.0")]
+		[UnsupportedOSPlatform ("tvos")]
```

Comment:
I see the message for obsolete ios 9, but I don't see the Unsupported for it

Reply:
Line 103. The diff has it broken up

Reply:
Wait, actually, what is going on?